File: Examples/Features/PageSetup/ConfigPrintTitleColumns.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.PageSetup
{
    public class ConfigPrintTitleColumns : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            var fileStream = this.GetResourceStream("xlsx\\PageSetup Print Title Columns.xlsx");
            workbook.Open(fileStream);
            IWorksheet worksheet = workbook.Worksheets[0];

            //Set columns to repeat at left
            worksheet.PageSetup.PrintTitleColumns = "$D:$G";

        }

        public override string TemplateName
        {
            get
            {
                return "PageSetup Print Title Columns.xlsx";
            }
        }
        public override string[] UsedResources
        {
            get
            {
                return new string[] { "xlsx\\PageSetup Print Title Columns.xlsx" };
            }
        }
    }
}

File: Examples.VB/Features/PageSetup/ConfigPrintTitleColumns.vb
Namespace Features.PageSetup
    Public Class ConfigPrintTitleColumns
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim fileStream = GetTemplateStream()
            workbook.Open(fileStream)

            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            'Set columns to repeat at left
            worksheet.PageSetup.PrintTitleColumns = "$D:$G"
        End Sub
        Public Overrides ReadOnly Property TemplateName As String
            Get
                Return "PageSetup Print Title Columns.xlsx"
            End Get
        End Property
    End Class
End Namespace
